### PR TITLE
created sentiment score attribute for comment

### PR DESCRIPTION
**Pull request by shivanighanta**
Use Sentiment API to generate and display a sentiment score for each comment

Link to Live Website: https://sghanta-step-2020-com2.appspot.com/

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -21,12 +21,15 @@
...
+      long id, String firstName, String lastName, String country, String subject, double score) {
     this.id = id;
     this.firstName = firstName;
     this.lastName = lastName;
     this.country = country;
     this.subject = subject;
+    this.score = score;
   }
-}
+}
```
Comment (jbir789) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:35]: nit: add a whitespace at the end of the file

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -104,6 +116,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/index.html");
   }
 
+  /** Run sentiment analysis and return sentiment score */
+  private double getSentimentScore(String message) throws IOException {
+    Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
```
Comment (jbir789) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:123]: Could this be a field variable, so it's only created once? Creating the service client is likely an expensive operation.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -115,4 +138,4 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
-}
+}
```
Comment (jbir789) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:141]: Can you add an empty line here as well?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (shivanighanta) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:123]: When I tried to make it a field variable I got this error: error: unreported exception java.io.IOException; must be caught or declared to be thrown

**Pull request merged**